shared/cfg/guest-os/Linux/Fedora/27.cfg

```diff
@@ -56,6 +56,10 @@
             kernel_params = "console=ttysclp0 serial"
             boot_path = images
             kernel = images/f27-s390x/kernel.img
+            # Anaconda hardcodes headless installation of F27 on s390x
+            vga = none
```

Comment:
Could you elaborate on a valid configuration (guest OS, test, etc) that should produce a valid VGA setup for s390x? I've tested with the boot test and while the devices are added:

```
...
MALLOC_PERTURB_=1  /usr/bin/qemu-system-s390x \
    -S  \
    -name 'avocado-vt-vm1' \
    -machine s390-ccw-virtio \
    -device virtio-mouse-ccw  \
    -nodefaults \
    -device virtio-gpu-ccw  \
...
```

And the test PASSes, the guest (JeOS.27) doesn't ever initialize the display.  Based on the commit message it sounds like Fedora 29 might support it?

Reply:
I'm sorry I haven't had time to try it on Fedora, but you can see attached image using RHEL.7.
![0011](https://user-images.githubusercontent.com/1074021/48426632-74ade000-e767-11e8-9e54-557f0a08e883.jpg)

Reply:
OK, I'll try with RHEL to properly review this PR.